addons/mrp/tests/test_order.py

```diff
@@ -1176,10 +1176,12 @@ def test_product_produce_11(self):
         mo_form.qty_producing = 3
         self.assertEqual(sum([x['quantity'] for x in mo_form.move_raw_ids._records]), 15, 'Update the produce quantity should change the components quantity.')
         mo = mo_form.save()
+        mo.move_raw_ids.picked = False
```

Comment:
I don't understand why this line should be added ?
This flow shouldn't be affected by your changes and when I run the test, the moves are indeed not picked